resource_booking_sale/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+An alternative implementation of sale_resource_booking (v12).
```

Comment:
This information is nice. But explain nothing on what the module does. At least a link to sale_resource_booking is welcome. :)

Reply:
fixed